Sources/TuistKit/Models/Product.swift

```diff
@@ -177,3 +184,15 @@ extension Product {
         }
     }
 }
+
+extension Product {
+    
+    var isStatic: Bool {
+        return [ .staticLibrary, .staticFramework ].contains(self)
+    }
+    
+//    var isTopLevelBundle: Bool {
```

Comment:
This is what we will require to determine if resources should be included. Need to decide if this work should live in this P/R or be developed as a separate P/R

Reply:
Agree with @kwridan, all the resources logic *(generated code, generation of bundles)* can be in a follow-up PR.